views/view.tt

```diff
@@ -374,6 +374,8 @@
                         
                         <div class="form-group ">
                           [%
+                            INCLUDE fields/hidden.tt id="refers_to_instance_id" value="10";
+                          
```

Comment:
I'm not quite following the need for this hidden field - is it needed and should it be hard-coded to a value of 10?

Reply:
It was originally needed to display the QueryBuilder element. The javascript trigger on the field has now changed, so it is possible that it can be removed. I will need to test this first

Reply:
From what i understand, in the JS layer, we currently only have one way of rendering the querybuilder (as a standardized component). It looks for this hidden field to render / process the element. I can change it, but it will take some time that i feel could be used better for other features / bugs. Would you like me to fix this first before we process the merge?

Reply:
Thanks Erik. I don't think there is any need to fix this now in entirety. I was just more concerned about the hard-coded value of "10". It looked like this might be a bug - can you confirm that this should indeed be a hard-coded constant value?